README.md

```diff
@@ -121,6 +121,24 @@ animator.start();
 animationView.cancelAnimation();
 ```
 
+You can add a color filter to the whole animation, a specific layer, or specific content within a layer:
+```java
+// Any class that conforms to the ColorFilter interface
+final SimpleColorFilter colorFilter = new SimpleColorFilter(Color.RED);
+
+// Adding a color filter to the whole view
+animationView.addColorFilter(colorFilter);
+
+// Adding a color filter to a specific layer
+animationView.addColorFilterToLayer("hello_layer", colorFilter);
+
+// Adding a color filter to specfic content on the "hello_layer"
+animationView.addColorFilterToContent("hello_layer", "hello", colorFilter);
+
+// Clear all color filters
+animationView.clearColorFilters();
+```
+Note: Color filters are only available for layers such as Image layer and Solid layer as well as content that includes fill, stroke, or group content. Shapes like Polystar or Rectangles are currently not supported.
```

Comment:
@seanfchan. This isn't accurate. The way AE and Lottie work is that they walk up the tree of content from bottom to top. When it gets to a DrawingContent (like fill, stroke, etc), it then continues up and collects all of the PathContents above it (like Shape, Polystar, Rectangle, Ellipse, etc) and draws them all. 

This isn't immediately obvious to somebody not familiar with after effects and one's instinct may be to provide the name of the path content they want to fill. However, that won't and shouldn't work (partly because you can have an arbitrary number of fills and strokes on the same path) and you should specify the name of the drawing content instead.

It's not super easy to grasp/explain so I just want to make sure the documentation helps devs/designers understand that before being frustrated that it doesn't work.